nil-docs/zkllvm/getting-started/installation.mdx

```diff
@@ -0,0 +1,266 @@
+import Tabs from '@theme/Tabs'
+import TabItem from '@theme/TabItem'
+
+# Installation and configuration
+
+## Using pre-built binaries
+
+The easiest way to access zkLLVM is to install pre-built binaries of its key components. 
+
+:::info
+
+All pre-built binaries discussed below are only supported on x86-x64 systems.
+
+:::
+
+### Install zkLLVM binaries
+
+zkLLVM is distributed as a Deb package which can be installed using the following commands.
+
+```bash
+echo 'deb [trusted=yes]  http://deb.nil.foundation/ubuntu/ all main' >>/etc/apt/sources.list
+apt update
+apt install -y zkllvm
+```
+
+This will install binaries for nearly all major components of zkLLVM such as `assigner` and the `clang` frontend. Note that the Rust frontend needs to be installed separately.
+
+### Install Rust binaries
+
+First, install `rustup`.
+
+```bash
+curl --proto '=https' --tlsv1.2 -sSf https://sh.rustup.rs | sh
+```
+
+Then, install `rslang` by executing the following command.
+
+```bash
+curl --proto '=https' --tlsv1.2 -sSf https://cdn.jsdelivr.net/gh/NilFoundation/zkllvm@master/rslang-installer.py | python - --channel nightly
+```
+
+Alternatively, download `rslang-installer.py` and run it.
+
+```bash
+curl --proto '=https' --tlsv1.2 -O https://cdn.jsdelivr.net/gh/NilFoundation/zkllvm@master/rslang-installer.py
+python rslang-installer.py --channel nightly
+```
+
+:::info[Standalone installation]
+
+To install `rslang` without `rustup`, run the below command.
+
+```bash
+curl --proto '=https' --tlsv1.2 -O https://cdn.jsdelivr.net/gh/NilFoundation/zkllvm@master/rslang-installer.py
+python rslang-installer.py --no-rustup --prefix PATH
+```
+
+:::
+
+Verify that the installation has been successful.
+
+```bash
+rustc +zkllvm -V
```

Comment:
One thing just came to my mind: maybe we should specify the output here. 

The thing is if installation for some reason went wrong, the "+zkllvm" toolchain won't be found and rustup may fallback to default one. So the user will see Rust version (default toolchain version), but it won't mean that everything is ok. 

Default toolchain version string looks like this:

```
$ rustc --version
rustc x.y.z (abcabcabc yyyy-mm-dd) 
```

If zkllvm rust is installed, this command will output this:

```
$ rustc +zkllvm --version
 rustc x.y.z (abcabcabc yyyy-mm-dd) (zkLLVM x.y.z) 
```

Note the zkLLVM version at the end - this is the marker of successful installation of zkllvm toolchain. 

Maybe we should mention this somehow.

Reply:
great idea, added